server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -449,6 +448,13 @@ public static void main(String[] args) throws Exception {
       log.info("SASL is not enabled, delegation tokens will not be available");
       delegationTokensAvailable = false;
     }
+    this.waitTimeBetweenScans =
+        aconf.getTimeInMillis(Property.MANAGER_TABLET_GROUP_WATCHER_INTERVAL);
```

Comment:
Computing it once at the time of the Manager startup makes this a fixed property, only changeable at restarts. You can do that, of course, but should add it to the `Property.fixedProperties`. However, a better idea is to have the field be a LongSupplier, and compute it from the config dynamically. You can use Guava's `Suppliers.memoize` to create the supplier. You can configure the memoizing supplier to cache the value for a certain amount of time, if you don't want it to do the lookup from the config too frequently.

Reply:
I think I can do this without the supplier. TabletGroupWatcher calls `manager.getWaitTImeBetweenScans()`. That method can just get the current value of the property from the configuration and return it.

Reply:
baea8d7 provides the TabletGroupWatcher with the current value of the property.